### PR TITLE
Made SwiftLint work with little changes

### PR DESCRIPTION
**Pull request by Ololopololo**
Check pls if it's okay to merge with main so we both will be able to pull main without any problems. This solution works fine for me

### CODE REVIEW

File: Outlay/.swiftlint.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 excluded:
   - Pods
+disabled_rules:
```
Comment (L1keMagic) [inline Outlay/.swiftlint.yml:3]: why do we need to disable this rule?

File: Outlay/Outlay.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -294,7 +294,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "${PODS_ROOT}/SwiftLint/swiftlint\n";
+			shellScript = "\"${PODS_ROOT}/SwiftLint/swiftlint\"\n";
```
Comment (L1keMagic) [inline Outlay/Outlay.xcodeproj/project.pbxproj:297]: why do we need additional slash here?

File: Outlay/Outlay.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -216,8 +216,8 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 2327AF70280C262B004CB7A0 /* Build configuration list for PBXNativeTarget "Outlay" */;
 			buildPhases = (
-				E539B782B35EB0303A48C136 /* [CP] Check Pods Manifest.lock */,
 				2376E12D280C2B19002B9336 /* Run SwiftLint */,
+				E539B782B35EB0303A48C136 /* [CP] Check Pods Manifest.lock */,
```
Comment (L1keMagic) [inline Outlay/Outlay.xcodeproj/project.pbxproj:220]: why did you switch the order of this components?

**Review by L1keMagic — CHANGES_REQUESTED** [review]

File: Outlay/Podfile.lock [hunk 4]
```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d45de82dcf86308ce89849c9103d3ec82aca5a74
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```
Comment (L1keMagic) [inline Outlay/Podfile.lock:20]: Could you please remove this

**Review by L1keMagic — CHANGES_REQUESTED** [review]

File: Outlay/.swiftlint.yml (hunk 1 above)
Comment (Ololopololo) [reply Outlay/.swiftlint.yml]: fixed

File: Outlay/Podfile.lock (hunk 4 above)
Comment (Ololopololo) [reply Outlay/Podfile.lock]: removed

**Review by L1keMagic — APPROVED** [review]
it looks good for me, approver

**Pull request merged**